### DISCUSSION TITLE
build production.dockerfile and run the image wont start the app

### DISCUSSION THREAD

**Discussion by accountunavaible** [ToolJet/ToolJet#5761] [General]
hi guys, I am new to Tooljet, and I love it, I am trying to build image for production.dockerfile, and run it, but seems it successfully built, but nothing start up. has any one face this issue? Thank you very much!

**Comment by adishM98** [comment]
Hi @accountunavaible ,

Is there any error that you get when trying to start the container ?

**Reply by accountunavaible** [reply]
thank you problem solved
